float64 startX
float64 startY
float64 goalX
float64 goalY
---
float64[] pathX
float64[] pathY
int64 throughObs
